library/Zend/Gdata/Analytics.php

```diff
@@ -55,12 +55,12 @@ public function __construct($client = null, $applicationId = 'MyCompany-MyApp-1.
     /**
      * Retrieve account feed object
      *
-     * @param string|Zend_Uri $uri
+     * @param string|Zend_Gdata_Analytics_AccountQuery $uri
      * @return Zend_Gdata_Analytics_AccountFeed|string
      */
     public function getAccountFeed($uri = self::ANALYTICS_ACCOUNT_FEED_URI)
     {
-        if ($uri instanceof Query) {
+        if ($uri instanceof Zend_Gdata_Analytics_AccountQuery) {
```

Comment:
`Query` class does not exist.  Sometimes I get the feeling that many components in ZF1 never actually worked. ;)